files/zh-cn/learn/css/css_layout/positioning/index.html

```diff
@@ -539,8 +541,9 @@ <h6 id="Sticky_positioning_example">Sticky positioning example</h6>
 }
 </pre>
 
+<div id="Sticky_2">
 <div class="hidden">
-<pre class="notranslate">body {
+<pre class="brush:css notranslate">body {
```

Comment:
I've added `brush:css` here. If this is missing, then the live sample won't work.